Gemfile.lock

```diff
@@ -6,49 +6,50 @@ PATH
       json-jwt (~> 1.11)
       jwt (~> 2.2)
       multi_json (~> 1.14)
-      omniauth (~> 1.8.1)
+      omniauth (~> 2.0.4)
```

Comment:
(not for this PR, but I'm just noticing now... we should remove Gemfile.lock from this repo and put it in gitignore, it's what the .gemspec is for)